scripts/cli.sh

```diff
@@ -0,0 +1,112 @@
+#! /bin/bash
+
+# A function to generate formatted .md files
+write_to_file() {
+    local cmd="$1"
+    local file="$2"
+    local program="$3"
+
+    # Remove first line of cmd to get rid of commit specific numbers.
+    cmd=${cmd#*$'\n'}
+
+    # We need to add the header and the backticks to create the code block.
+ printf "# %s\n\n\`\`\`\n%s\n\`\`\`" "$program" "$cmd" > "$file"
```

Comment:
Seems to be missing an indent
```suggestion
     printf "# %s\n\n\`\`\`\n%s\n\`\`\`" "$program" "$cmd" > "$file"
```